python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, log=None, doStripHeader=False):
         self.log = log
         self.doStripHeader = doStripHeader
 
-    def __call__(self, md, exposureId):
+    def __call__(self, md, exposureId=0, id=0):
```

Comment:
I think the only code that calls this now is in obs_test (in addition to a test of it in obs_base) so we may want to consider deleting it completely now. The issue is that it's adding a new optional parameter `id` that makes it incompatible with the ObservationInfo implementation (which already ignores the parameter). If you retain it I think you are going to have to add `id` to the other one as well (and maybe issue a warning if you get given it).  Alternatively, don't add this parameter here at all and always set it to 0 inside because there isn't a situation where people are going to be passing it in.

Reply:
This is a good point: I've reverted this commit. DM-13943 will deprecate exposureId, and we can make it an optional kwarg then.